test.js

```diff
@@ -0,0 +1,21 @@
+import test from 'ava';
+import m from 'alfy-test';
```

Comment:
Should be `m`=>`alfyTest`. As in this case it's not importing the main module `m`.